sand/extensions/search/default_search.py

```diff
@@ -0,0 +1,72 @@
+from typing import List
+import re
+from sand.extension_interface.search import IEntitySearch, IOntologySearch
+from sm.misc.funcs import import_attr
+from sand.models.search import SearchResult
+from sand.config import SETTINGS
+
+
+class DefaultSearch(IEntitySearch, IOntologySearch):
+
+    def __init__(self):
+        self.default_classes = import_attr(SETTINGS['ont_classes']['default'])
+        self.default_entities = import_attr(SETTINGS['entity']['default'])
+        self.default_properties = import_attr(SETTINGS['ont_props']['default'])
+
+    def local_search(self, default_entities: dict, search_text: str) -> List:
+        """ performs local partial text search across default entities"""
+        query_tokens = re.findall("[a-zA-Z]+", search_text)
+        search_results = []
+        for key, entity in default_entities.items():
+            label = entity.label.lower()
+            if all(list(map(lambda x: x.lower() in label, query_tokens))):
```

Comment:
you can still move the last .lower call outside

Reply:
I have used map to remove the lower call in the bottom. There are several ways to implement this, please let me know if you have anything specific in mind. 

I was thinking to use list comprehensions as well as an alternative solution.